lib/table.js

```diff
@@ -32,6 +32,9 @@ function Table (name) {
       column.name = name
       column.nullable = options.nullable
       column.primary = options.primary
+      if (options.length) {
```

Comment:
```suggestion
      if (options.hasOwnProperty('length')) {
```

Reply:
Do we want to check for a truthy value too?